contracts/deployment_scripts/testing/001_gas_testing_deployment.ts

```diff
@@ -4,6 +4,7 @@ import { Receipt } from 'hardhat-deploy/dist/types';
 
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+    return;
```

Comment:
you mean to commit that?

Reply:
I should probably delete this at some point. It's never ran, just for debugging purposes against Moray's tests.